File: Smartsense.Mobile.Android/app/src/main/java/com/smartsense/covid/adapters/companionAdapter/Companion.java
package com.smartsense.covid.adapters.companionAdapter;

import androidx.annotation.Keep;

@Keep
public class Companion {
    private long userid;
    private String name;
    private String surname;
    private String email;

    public long getUserid() {
        return userid;
    }

    public void setUserid(long userid) {
        this.userid = userid;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSurname() {
        return surname;
    }

    public void setSurname(String surname) {
        this.surname = surname;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    @Override
    public String toString() {
        return "Companion{" +
                "id=" + userid +
                ", name='" + name + '\'' +
                ", surname='" + surname + '\'' +
                ", email='" + email + '\'' +
                '}';
    }
}

File: Smartsense.Mobile.iOS/covid/View/CompanionViews/Companion.swift
//
//  Companion.swift
//  covid
//
//  Created by SmartSense on 12.08.2021.
//

import Foundation

struct Companion: Hashable, Codable, Identifiable{
    var id: Int64
    var name: String
    var surname: String
    var email: String
    var password: String
    
    init(id: Int64, name: String, surname: String, email: String, password: String) {
        self.id = id
        self.name = name
        self.surname = surname
        self.email = email
        self.password = password
    }
}
